semgrep/tests/e2e/snapshots/test_rule_parser/test_rule_parser__failure__error_messages/badpattern/error-in-color.txt

```diff
@@ -1,2 +1,6 @@
 running 1 rules...
-[31mRule id: eqeq-is-bad contains a pattern that could not be parsed as a pattern for language Python: `$X == $X 3`[39m
+[31msemgrep error[39m: invalid pattern
+  --> semgrep temp file:5
```

Comment:
Was the name of the file `temp` or is it in fact a temporary file? If we get temporary files then it may be better not to include them in the report for now.